html/viewer/index.html

```diff
@@ -3,6 +3,15 @@
 <head>
   <title>Firefox Icons</title>
   <link href="css/app.css" rel="stylesheet"/>
+  <script>
+    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+    })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+
+    ga('create', 'UA-100552286-1', 'auto');
+    ga('send', 'pageview');
+  </script>
```

Comment:
So, I agree with the putting-it-at-the-bottom part, but the async and defer are [a little more complicated](https://github.com/h5bp/html5-boilerplate/pull/1696#issuecomment-96118051), so let's hold off on those for now.